app/uk/gov/hmrc/voapropertylinking/connectors/modernised/MdtpDashboardManagementApi.scala

```diff
@@ -45,7 +45,12 @@ class MdtpDashboardManagementApi @Inject()(
 
   def getAssessment(authorisationId: Long)(implicit hc: HeaderCarrier): Future[Option[PropertiesView]] = {
     val url = baseUrl + s"/mdtp-dashboard-management-api/mdtp_dashboard/view_assessment?listYear=$listYear&authorisationId=$authorisationId"
-    http.GET[Option[PropertiesView]](url).map(_.map(_.upperCase)) recover { case _: NotFoundException => None }
+
+    http.GET[Option[PropertiesView]](url).map { optPropertiesView =>
+      PartialFunction.condOpt(optPropertiesView) {
```

Comment:
does this compile??? from the look of it, it looks like itll be a future option of an option

Reply:
i am so puzzled by this. I think this has gone wrong in the merge, i clicked the magic wand and this happened. I am going to sort this out

Reply:
yes, this definitely is a bad merge issue... thanks for paying attention 👍